assets/src/app.js

```diff
@@ -220,15 +231,24 @@ function App() {
     setFilterParams({ ...newFilterParams, ...capacity });
   }, [capacityFilter]);
 
-  // Set resource filter.
+  // Set facility filter.
   useEffect(() => {
-    const resourceValues = resourceFilter.map(({ value }) => value);
+    delete filterParams.monitorEquipment;
```

Comment:
Instead of deleting in state object directly it is best practice to "clone the object" -> "mutate it" -> setFilterParams(clonedObject).